content/en/news/2019/announcing-1.4/change-notes/index.md

```diff
@@ -16,6 +16,9 @@ weight: 10
 ## Security
 
 - **Added** the [`v1beta1` authorization policy model](/blog/2019/v1beta1-authorization-policy/) for enforcing access control. This will eventually replace the [`v1alpha1` RBAC policy](/docs/reference/config/security/istio.rbac.v1alpha1/).
+- **Added** experimental support for [automatic mutual TLS](docs/tasks/security/authentication/auto-mtls/).
```

Comment:
can we add a brief explanation, `to enable mutual TLS without need of configuring destination rule`?

Customers experiencing the pain of the destination rule's configuration can have some sense right away.